app/views/contactPreference/AddEmailAddressView.scala.html

```diff
@@ -19,46 +19,74 @@
 @import forms.YesNoForm._
 @import views.html.templates.errors.ErrorSummary
 
-@this(mainTemplate: MainTemplateOriginal, radioGroup: RadioGroup, errorSummary: ErrorSummary, formWithCSRF: FormWithCSRF)
+@this(mainTemplate: MainTemplate,
+    formWithCSRF: FormWithCSRF,
+    govukBackLink: GovukBackLink,
+    govukErrorSummary : GovukErrorSummary,
+    govukInsetText : GovukInsetText,
+    govukRadios : GovukRadios,
+    govukButton: GovukButton)
 
 @(form: Form[YesNo])(implicit user: User[_], messages: Messages, appConfig: config.AppConfig)
 
-@furtherContent = {
-    <div>
-        <p id="form-hint"> @messages("cPrefAddEmail.line1")</p>
-        <p> @messages("cPrefAddEmail.line2")</p>
-
-        <p>
-            <strong class="bold-small">
-                @messages("cPrefAddEmail.question")
-            </strong>
-        </p>
-    </div>
-}
-
 @mainTemplate(
-    title = if(form.errors.nonEmpty) messages("common.error.prefixTitle", messages("cPrefAddEmail.title"))
+    pageTitle = if(form.errors.nonEmpty) messages("common.error.prefixTitle", messages("cPrefAddEmail.title"))
     else messages("cPrefAddEmail.title")) {
 
-    <a class="link-back" href="@controllers.contactPreference.routes.EmailPreferenceController.show().url">@messages("base.back")</a>
+    @govukBackLink(BackLink(
+        href = controllers.contactPreference.routes.EmailPreferenceController.show().url,
+        content = Text(messages("base.back"))
+    ))
 
-    @errorSummary(messages("common.error.heading"), form)
+    @if(form.hasErrors) {
+        @govukErrorSummary(ErrorSummary(
+                errorList = Seq(
+                ErrorLink(
+                href = Some("#yes_no"),
+                content = Text(messages("cPrefAddEmail.error"))
+            )),
+                title = Text(messages("common.error.heading"))
+            ))
+    }
+
+    <h1 class="govuk-heading-l">@messages("cPrefAddEmail.title")</h1>
+
+    <div>
+        <p class="govuk-body"> @messages("cPrefAddEmail.line1")</p>
+        <p class="govuk-body"> @messages("cPrefAddEmail.line2")</p>
+    </div>
 
     @formWithCSRF(action = controllers.contactPreference.routes.AddEmailAddressController.submit, 'novalidate -> "novalidate") {
```

Comment:
Can you remove the `novalidate` parameter here as you get this by default in the `govuk-frontend` `FormWithCSRF`